test_helpers_end_to_end/src/snapshot_comparison.rs

```diff
@@ -136,6 +136,14 @@ fn make_output_path(input: &Path) -> Result<PathBuf, OutputPathError> {
     // go one level down (from parent dir to out-dir)
     out.push("out");
 
+    // make best effort attempt to create output directory if it
+    // doesn't exist (it does not on a fresh checkout)
+    if !out.exists() {
+        if let Err(e) = std::fs::create_dir(&out) {
+            eprintln!("WARNING: could not create output directory {out:?}: {e}");
```

Comment:
Hm, if we aren't able to create the directory, shouldn't we panic rather than warning and attempting to continue?

Reply:
What happened on CI was that the subsequent test failed with `'can't write to output file <filename>'` which was pretty clear.

I figured that a warning here would be a sufficient clue when the test subsequently failed, but after some more thought, I think failing fast is a better idea. Thank you for the suggestion

```suggestion
            panic!("Could not create output directory {out:?}: {e}");
```